### DISCUSSION TITLE
Guide for google authentication

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#2104] [Questions] [answered]
Is there any guide on how to get client id and secret for google auth?

**Comment by harshcut** [comment, marked as answer]
There is a guide section coming soon ([auth-google](https://github.com/supabase/supabase/blob/master/web/docs/guides/auth/auth-google.mdx)). By then you could watch [Supabase Auth Deep Dive](https://www.youtube.com/watch?v=_XM9ziOzWk4).
